src/Components/RightSidebar/RequestSignatureTab.vue

```diff
@@ -58,6 +58,10 @@
 				@click="validationFile()">
 				{{ t('libresign', 'Validate') }}
 			</NcButton>
+			<NcButton variant="primary"
```

Comment:
Only a button need to be primary.

https://nextcloud-vue-components.netlify.app/#/Components/NcButton